Training/Program.cs

```diff
@@ -31,9 +31,9 @@
       return "0";
    string hexChars = "0123456789ABCDEF";                                   //hexadecimal characters are defined in a string as hexChars
    string hex = "";
-   while (dec > 0) {                                                                       
-      hex = hexChars[dec % 16] + hex;                                      //empty string hex is updated by dividing the decimal number by 16 using modulo operator,
-      dec = dec / 16;                                                      //after which the quotient indicates the index of the hexChars string
+   while (dec > 0) {
+      hex+= hexChars[dec % 16];                                            //empty string hex is updated by dividing the decimal number by 16 using modulo operator,
+      dec/= 16;                                                            //after which the quotient indicates the index of the hexChars string

```

Comment:
There should be a gap between the number and the operator: _hex += hexChars[dec % 16]_

The function naming is not maintained. The name should start with a capital letter.